tests/unit/lifecycle/test_lifecycle.py

```diff
@@ -385,77 +385,6 @@ def test_non_prime_and_no_version(self):
         lifecycle.execute(steps.PULL, project_config)
 
 
-class DirtyBuildScriptletTestCase(LifecycleTestBase):
```

Comment:
This is actually quite an important test, can you just change it to use the override scriptlets instead?

Reply:
sure, I just saw old methods and decided to wipe. good catch

Reply:
wait, these same tests already exist